msb/network/serial/publisher.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import annotations
+import serial
+
+from .config import SerialConf
+from msb.network.types import Publisher
+
+
+def serial_packer(input_dict) -> bytes:
+    return ",".join(input_dict.values())
+
+
+class SerialPublisher(Publisher):
+    def __init__(self, config: SerialConf):
+        self.config = config
+        self.packer = serial_packer
+        self.connect()
+
+    def connect(self):
+        self.serial: serial.Serial = serial.Serial(
+            port=self.config.port,
+            baudrate=self.config.baudrate,
+            bytesize=self.config.bytesize,
+            parity=serial.PARITY_NONE,
```

Comment:
Parity is defined in SerialConf but not used here.

Reply:
Removed from config.